### DISCUSSION TITLE
Admin Panel - URLs Found but No Listings Show Up

### DISCUSSION THREAD

**Discussion by thehoffm** [YOURLS/YOURLS#3338] [Q&A]
I'm having an issue where I can create and search for URLs from the admin dashboard and while the dashboard recognizes that URLs exist, nothing is displayed on the page:  

![image](https://user-images.githubusercontent.com/105566590/168496515-34037b73-5b42-47e7-8084-4606eba55632.png)

Any ideas?

**Comment by dgw** [comment]
Browser extension (element filtering) gone rogue?

View the page source (Ctrl-U in most browsers) to see if the shortlinks are present there. Heck, try a different browser that you don't normally use.

**Reply by thehoffm** [reply]
That was my first thought - Chrome with Disabled Extensions, Firefox, even got me to open Edge for the first time in years. No dice.

**Reply by ozh** [reply]
Does this happen for any searched term ?

**Reply by thehoffm** [reply]
@ozh Any searched term. I added another link and searched for it just to confirm. Same result.

**Reply by ozh** [reply]
No idea. Going for the obvious : you sure it's not a problem of case sensitiveness ?

**Reply by thehoffm** [reply]
Nope. I made a link to test that's just /cnn that goes to cnn.com - searched for cnn:
![image](https://user-images.githubusercontent.com/105566590/168668289-15970bc8-feab-4fdf-bc04-99650f5db259.png)

You can see it finds the url but doesn't display anything.

Random idea: Could it be a matter of permissions somewhere in pulling the results for the database?

**Reply by ozh** [reply]
The thing is, it does pull stuff from the DB, hence the "display 1 to 1 of 1 URLs" (nice wording btw, who made this??? :) so after counting why wouldn't the display work? I don't know.

What *I\* would do : add a bunch of var_dump after https://github.com/YOURLS/YOURLS/blob/master/admin/index.php#L293 to see what vars are empty when they should not ¯\\\_(ツ)_/¯

**Reply by thehoffm** [reply]
@ozh I made a hail mary attempt and just installed every PHP extension package that wasn't already installed via Apache. Things finally started to show up. I appreciate the help.

**Reply by dgw** [reply]
@thehoffm Could you tell me which ones weren't installed? I'm working on documenting the extensions YOURLS needs and the list could help me build that out.

**Reply by thehoffm** [reply]
@dgw I can take a guess for you what was installed vs what wasn't based off of the configuration of the other packages on other versions I had that were consistent.

Extensions I believe were installed before:
pear
php-cli
php-common
php-devel
php-ftp
php-litespeed
php-mysqlnd
php-pdo
php-posix
php-xml
runtime

After:
libc-client
pear
php-cli
php-common
php-curl
php-dba
php-dbg
php-devel
php-enchant
php-exif
php-fileinfo
php-fpm
php-ftp
php-gd
php-gettext
php-gmp
php-iconv
php-imap
php-intl
php-ldap
php-litespeed
php-mbstring
php-memcached
php-mysqlnd
php-odbc
php-opcache
php-pdo
php-pgsql
php-posix
php-process
php-pspell
php-snmp
php-soap
php-sockets
php-sodium
php-sourceguardian
php-tidy
php-xml
php-zip
runtime
scldevel

**Reply by dgw** [reply]
iconv, mbstring, and curl are the ones that jump out, based on the list I already have. Might be worth me going in and verifying that the ones marked optional really are… Thanks!
